python/tests/test_semantics.py

```diff
@@ -75,6 +75,34 @@ def test_origin(f):
     assert random_origin.file_set_nr == 1042
     assert random_origin.file_nr     == 6
 
+def test_axis(f):
```

Comment:
Btw, I still believe that core tests on actual files might be beneficial. Maybe there is no need to test every single attribute, but checking number of objects or whatever might still provide some value.

Reply:
I have looked through all of the test files, and none of them contains objects such as long_name, axis or zone! Maybe we can get some additional testfiles in the future